src/unit_tests/wazuh_db/test_wdb_upgrade.c

```diff
@@ -105,15 +105,18 @@ void test_wdb_upgrade_global_table_fail(void **state)
 }
 
 void test_wdb_upgrade_global_update_success(void **state)
-{   
+{
     wdb_t *ret = NULL;
     test_struct_t *data  = (test_struct_t *)*state;
-    
+
     expect_string(__wrap_wdb_metadata_table_check, key, "metadata");
     will_return(__wrap_wdb_metadata_table_check, 0);
     expect_string(__wrap__mdebug2, formatted_msg, "Updating database '000' to version 1");
     expect_string(__wrap_wdb_sql_exec, sql_exec, schema_global_upgrade_v1_sql);
     will_return(__wrap_wdb_sql_exec, 0);
+    expect_string(__wrap__mdebug2, formatted_msg, "Updating database '000' to version 2");
```

Comment:
This is previous to this changes (line 47), but wdb->id as "000" probably was wrongly inherited from other tests

Reply:
Updated with **global** !